lib/Cron/CronHelper.php

```diff
@@ -19,20 +18,29 @@ class CronHelper {
   // Error codes
   const DAEMON_EXECUTION_LIMIT_REACHED = 1001;
 
-  static function getDaemonExecutionLimit() {
-    $wp = Functions::get();
-    $limit = $wp->applyFilters('mailpoet_cron_get_execution_limit', self::DAEMON_EXECUTION_LIMIT);
+  /** @var SettingsController */
+  private $settings;
+
+  /** @var WPFunctions */
+  private $wp;
+
+  public function __construct(SettingsController $settings, WPFunctions $wp) {
```

Comment:
Just one question - can some of the methods in this class be made `private`? Or do we need all of them `public`?

Reply:
Currently they all need to be public: all but two are referenced in other classes, and the two remaining (`queryCronUrl`, `getSiteUrl`) are used in tests.